src/handlers/PanGestureHandler.ts

```diff
@@ -150,7 +150,9 @@ export interface PanGestureHandlerProps
    * to 0. If only one number `p` is given a range of `(-inf, p)` will be used
    * if `p` is higher or equal to 0 and `(-p, inf)` otherwise.
    */
-  activeOffsetY?: number | number[];
+  activeOffsetY?:
+    | number
+    | [activeOffsetYStart: number, activeOffsetYEnd: number];
```

Comment:
Can you also update `panGesture.ts`?

Reply:
Sure, done in [5066678](https://github.com/software-mansion/react-native-gesture-handler/pull/2872/commits/50666789265a7908a2565ebc801d883d9ccefb59)